lib/internal/event_target.js

```diff
@@ -482,6 +482,10 @@ class EventTarget {
     }
     type = String(type);
 
+    if (signal === null) {
```

Comment:
This check seems a bit too specific, we should probably throw on many other values, e.g.:

```js
new EventTarget().addEventListener('foo', () => {}, { signal: {} }) // should throw
```

Instead, I think we should use `validateAbortSignal` from `internal/validators`.